dbos/_dbos.py

```diff
@@ -172,6 +172,8 @@ def register_wf_function(self, name: str, wrapped_func: F, functype: str) -> Non
         if name in self.function_type_map:
             if self.function_type_map[name] != functype:
                 raise DBOSConflictingRegistrationError(name)
+            if name != TEMP_SEND_WF_NAME:
+                dbos_logger.warning(f"Duplicate registration for function '{name}'")
```

Comment:
Duplicate registration of function {name}. A function named {name} has already been registered with DBOS. Please ensure all registered functions have unique names.